int32 ALLOWABLE_MOTION_POLYTOPE=1
int32 CONSTRAINED_ALLOWABLE_MOTION_POLYTOPE=2
int32 VELOCITY_POLYTOPE=3
int32 CONSTRAINED_VELOCITY_POLYTOPE=4


int32 polytope_type
sensor_msgs/JointState[] sampled_joint_states # Configuration at which to calculate constraints at 
bool show_polytope
---
constrained_manipulability/PolytopeHyperplanes[] polytope_hyperplanes

